js/tests/browsers.js

```diff
@@ -51,13 +51,6 @@ const browsers = {
     browser: 'Firefox',
     browser_version: 'latest'
   },
-  EsrWin10: {
```

Comment:
This one was only in Boosted. I chose to remove it to have the same file than Bootstrap.

Reply:
But ESR is the one used inside Orange 🤔

Reply:
OK: verified in the history of both Boosted and Bootstrap. Thanks for the info, I've put it back in this PR surrounded by a `Boosted mod` comment to highlight it via 45207e3c0b0bae8c76689c911efdff836e5eb8dc